testgres/api.py

```diff
@@ -7,27 +7,27 @@
 edit configuration files, start/stop cluster, execute queries. The
 typical flow may look like:
 
->>> with get_new_node() as node:
+>>> with get_new_node('test') as node:
```

Comment:
IMHO it's better to have at least one example with no args at all. It would show user that literally any arg of `get_new_node()` has a decent default.

Reply:
Fair enough